### PR TITLE
Issue #17 - Levenshtein w/ domain whitelist

### PR DESCRIPTION
**Pull request by 409H**
See: https://github.com/409H/EtherAddressLookup/pull/25

* Checks to see if the domain is in the whitelist - it if is, it quits and lets you interact with the web page.
* If the domain isn't in the whitelist, then it'll check the blacklist and run a levenshtein algo on the domain against [www.]myetherwallet - if the domains are too similar, it will block interaction with the page.

### CODE REVIEW

File: js/options.js [hunk 1]
```diff
@@ -103,4 +133,23 @@ function getBlacklistedDomainsFromSource()
         }
     }
     return {"timestamp":0,"domains":[]};
+}
+
+function getWhitelistedDomainsFromSource()
+{
+    console.log("Getting whitelist from GitHub now");
+    var objAjax = new XMLHttpRequest();
+    objAjax.open("GET", "https://raw.githubusercontent.com/409H/EtherAddressLookup/master/whitelists/domains.json", true);
```
Comment (sogoiii) [inline js/options.js]: You haven't added this file yet.

Comment (sogoiii) [inline js/options.js]: Ah, I see did the before this commit. by bad.

Comment (409H) [inline js/options.js]: Already pushed that file to `master` :) https://github.com/409H/EtherAddressLookup/commit/bbf7f54f1cdd1d094305c1f47f1c5cc166b79ed5

Comment (409H) [inline js/options.js]: Awesome, no worries :)

**Comment by 409H** [comment]
**New tag:** https://github.com/409H/EtherAddressLookup/releases/tag/v1.3.6

**Pull request merged**